updater/internal/updater/updater.go

```diff
@@ -16,10 +16,12 @@
 package updater
 
 import (
```

Comment:
3. Upgrading a system using the updater, after upgrading the system from step 1
- I updated a 1.79.2 agent to the 1.80-alpha.0-SNAPSHOT version using the package install with --file flag
- This worked but trying to use the update button to 1.80-alpha.1-SNAPSHOT hit the same behavior as above, where it hangs with no logs.

Reply:
Okay, this invalidates my previous statement. ill look into it.